mautrix_telegram/portal/base.py

```diff
@@ -456,6 +457,13 @@ def get_by_entity(cls, entity: Union[TypeChat, TypePeer, TypeUser, TypeUserFull,
                                receiver_id if type_name == "user" else entity_id,
                                type_name if create else None)
 
+    @classmethod
+    async def reached_portal_limit(cls) -> bool:
+        limit = config.get("bridge.max_portal_rooms", 0)
+        if limit == 0:
```

Comment:
The comments in the sample config suggest that this should be -1.
limit == 0 should always return `True` for the limit being reached, I guess.

```suggestion
        if limit == -1:
```